ClientCore/ClientConfiguration.cs

```diff
@@ -278,23 +278,26 @@ private List<TranslationGameFile> ParseTranslationGameFiles()
         {
             List<TranslationGameFile> gameFiles = new();
 
-            for (int i = 0; clientDefinitionsIni.KeyExists(TRANSLATIONS, $"GameFile{i}"); i++)
+            foreach (string key in clientDefinitionsIni.GetSectionKeys(TRANSLATIONS))
             {
-                // the syntax is GameFileX=path/to/source.file,path/to/destination.file[,checked]
-                string value = clientDefinitionsIni.GetStringValue(TRANSLATIONS, $"GameFile{i}", string.Empty);
-                string[] parts = value.Split(',', StringSplitOptions.RemoveEmptyEntries).Select(s => s.Trim()).ToArray();
+                // syntax must bet GameFileX=..., where X - any text.
+                if (!key.StartsWith("GameFile"))
+                    continue;
+
+                string   value = clientDefinitionsIni.GetStringValue(TRANSLATIONS, key, string.Empty);
+                string[] parts = value.Split(',');
```

Comment:
```suggestion
                string value = clientDefinitionsIni.GetStringValue(TRANSLATIONS, key, string.Empty);
                string[] parts = value.Split(',');
```
I see the idea in the style change but I think there's more harm in this being inconsistent with the rest of client codebase than in the subjective appeal of lining up the variable declarations in this specific place. If needed such changes must be made after discussion with client maintainers/contributors and across the client globally.

Reply:
Ok, i'll keep that in mind.